solutions/dmoj-Phidias.mdx

```diff
@@ -0,0 +1,83 @@
+---
+id: dmoj-phidias
+source: DMOJ
+title: Phidas
+author: Jesse Choe
+---
+
+Let $\texttt{dp}[i][j]$ be the minimum total wasted area with a rectangular slab size of $i\times j$. Initialize $\texttt{dp}[i][j]$ as $i \cdot j$ and set $\texttt{dp}[i][j]$ as 0, where $x_i, y_i$ are the width and heights of the desired plate sizes, respectively. Check every sub-rectangle with either $i$ or $j$ fixed (but not both) and store the minimum total area that must be wasted for the optimal sub-rectangle.
```

Comment:
should it be `set $\texttt{dp}[x_i][y_i]$ as 0`?

Reply:
Yes.

Reply:
err what you put was `set $\texttt{dp}[i][i]$ as 0`, so this is the suggestion (hopefully that's right):
```suggestion
Let $\texttt{dp}[i][j]$ be the minimum total wasted area with a rectangular slab size of $i\times j$. Initialize $\texttt{dp}[i][j]$ as $i \cdot j$ and set $\texttt{dp}[x_i][y_i]$ as 0, where $x_i, y_i$ are the width and heights of the desired plate sizes, respectively. Check every sub-rectangle with either $i$ or $j$ fixed (but not both) and store the minimum total area that must be wasted for the optimal sub-rectangle.
```